MapleServer2/PacketHandlers/Game/RequestTaxiHandler.cs

```diff
@@ -17,21 +17,30 @@ class RequestTaxiHandler : GamePacketHandler
         public override void Handle(GameSession session, PacketReader packet)
         {
             byte mode = packet.ReadByte();
-            int mapId = packet.ReadInt();
 
             switch (mode)
             {
-                case 3: // rotors using mesos?
+                case 3: // rotors using mesos
+                    session.Player.Mesos -= 60000;
```

Comment:
There should probably be a check if the player has those mesos before trying to take them

Reply:
done